### ISSUE TITLE
No superclass method `distance_of_time_in_words' error when using version 5.0.0

### ISSUE THREAD

**Issue by denisahearn** [radar/distance_of_time_in_words#110]
After upgrading to version 5.0.0 of this gem, my Rails app started experiencing this issue:

```
NoMethodError:
 super: no superclass method `distance_of_time_in_words' for #<ActionView::Base:0x00007fca8aa2c540>
 Did you mean?  distance_of_time_in_words_to_now
                distance_of_time_in_percent
                _distance_of_time_in_words
# /Users/my_user/.rvm/gems/ruby-2.6.5@cloudvault/gems/dotiw-5.0.0/lib/dotiw/action_view/helpers/date_helper.rb:13:in `distance_of_time_in_words'
```

If I lock my Gemfile at version 4.0.1 of this gem then the issue goes away.

The code in my Rails app that is calling `distance_of_time_in_words` helper is calling it in a controller this way:

```
ActionController::Base.helpers.distance_of_time_in_words(...)
```

The version of Rails that my app is using is 6.0.3.2.

Is there something I need to do in my app to make this helper method work with version 5.0.0, or is this a regression issue with this gem version?

**Comment by denisahearn** [comment]
Please let me know if you would like me to submit a sample Rails app that demonstrates this issue, and I will put one together and upload it here.

**Comment by dblock** [comment]
It's probably a regression, calling `ActionController::Base.helpers.distance_of_time_in_words` invokes a class method and so we don't have `super` in https://github.com/radar/distance_of_time_in_words/blob/master/lib/dotiw/action_view/helpers/date_helper.rb#L13, what we're really trying to call here is `DOTIW::Methods.distance_of_time_in_words`.

Instead of using `ActionController::Base.helpers.distance_of_time_in_words` do what the README says [here](https://github.com/radar/distance_of_time_in_words#rails), and call `distance_of_time_in_words` directly. I think this is going to work.

If you want to help, add a spec for this scenario [here](https://github.com/radar/distance_of_time_in_words/blob/master/spec/lib/dotiw_spec.rb#L326). Just call `ActionController::Base.helpers.distance_of_time_in_words`, which should fail the same way. Let's try to fix it, I suspect changing `super` to `DOTIW::Methods.distance_of_time_in_words` on that line will fix it and pass all the existing specs. Make a PR.

**Comment by denisahearn** [comment]
@dblock Here is a PR that fixes the issue: https://github.com/radar/distance_of_time_in_words/pull/111. Looking forward to your feedback.

**Comment by dblock** [comment]
Fixed un https://github.com/radar/distance_of_time_in_words/pull/111

**Comment by dblock** [comment]
I released 5.2.0, thanks for your help @denisahearn! If you want to do more work I think it'd be nice to break up the specs a bit separating Rails vs. non-Rails into separate files, etc.
